src/dangerRender.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+// Provides dev-time typing structure for  `danger` - doesn't affect runtime.
+// https://github.com/danger/danger-js/blob/main/docs/usage/extending-danger.html.md#writing-your-plugin
+import { GenericTicketType, I_GenericTicket } from "./types";
+export declare function warn(message: string): void;
+export declare function markdown(message: string): void;
+export declare function schedule<T>(asyncFunction: Promise<T>): void;
+
+const StoryEmoji: Record<GenericTicketType, string> = {
+  chore: "⚙️",
+  epic: "⚡",
+  feat: "🌟",
+  fix: "🐞",
+};
+
+export const printWarningError = (e: Error): void => {
```

Comment:
I'd avoid this kind of wrapper methods which add little value (just use `warn(e.message)` when needed)